charts/camunda-platform/templates/_helpers.tpl

```diff
@@ -142,3 +142,10 @@ Set imagePullSecrets according the values of global, subchart, or empty.
     {{- .Values.operate.contextPath -}}
   {{- end -}}
 {{- end -}}
+
+{{ define "camundaPlatform.connectorsURL" }}
```

Comment:
I'd suggest removing it since it's not used, and it's easy to add it back when it's needed.